apps/games/g_snake/main.py

```diff
@@ -0,0 +1,110 @@
+from ui import Canvas
```

Comment:
The `g_2048` name was only used for 2048 because a Python module name can't start with a letter, so you can safely rename `g_snake` to just `snake` =)